guides/09_Amazon_Voice_Focus.md

```diff
@@ -285,7 +285,23 @@ A specification is used to derive a runtime _configuration_ when a transformer i
 
 Enabling Voice Focus with Echo Reduction is a two step process: 
 
-### 1. Creating a meeting with support for Echo Reduction
+### 1. Create a meeting with support for Echo Reduction using Amazon Chime Meeting SDK namespace
+
+To create meetings, the following line of code is used to address the namespace:
```

Comment:
I think we can make this more clear:

Use Amazon Chime Meetings SDK to create a meeting with echo reduction feature. If you are using `AWS.Chime` namespace to create the Chime SDK meetings, you have to migrate to use the `AWS. ChimeSDKMeetings` namespace.

Before:
```js
const chimeMeetings = AWS.Chime();
```

After:
```js
const chimeMeetings = AWS.ChimeSDKMeetings({ region: "eu-central-1" });
```

Reply:
I like @simmkyu's version better, request to check his comment: https://github.com/aws/amazon-chime-sdk-js/pull/1848#discussion_r763547733